README.md

```diff
@@ -1 +1,158 @@
-# Project-code-review
\ No newline at end of file
+
+
+<a name="readme-top"></a>
+
+
+<div align="center">
+  
+  <h3><b> Project-code-review</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Project-code-review <a name="about-project"></a>
+
+**Project-code-review** is a repository.The goal is to master all of the tools and best practices learned in previous steps.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/css/">css</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Demo of Linters**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+[comming soon]
+
+### Setup
+
+Clone this repository to your desired folder:
+
+[comming soon]
+
+### Install
+
+Install this project with:
+
+[comming soon]
+
+### Usage
+
+To run the project, execute the following command:
+
+[comming soon]
+
+### Run tests
+
+To run tests, run the following command:
+
+[comming soon]
+
+### Deployment
+
+You can deploy this project using:
```

Comment:
I suggest that you complete your readme section. It's important to provide clear and concise information about your project or application, so users can understand what it does and how to use it. 📝😊

Reply:
- [ ] This is good. your comment is clear & polite 😄